packages/turf-clone/index.js

```diff
@@ -70,12 +77,21 @@ function cloneFeature(feature) {
  * @returns {FeatureCollection<any>} cloned Feature Collection
  */
 function cloneFeatureCollection(geojson) {
-    return {
-        type: 'FeatureCollection',
-        features: geojson.features.map(function (feature) {
-            return cloneFeature(feature);
-        })
-    };
+    const cloned = {type: 'FeatureCollection'};
+    // Add id & bbox first
+    if (geojson.id) cloned.id = geojson.id;
+    if (geojson.bbox) cloned.bbox = geojson.bbox;
+
+    // Custom Properties
+    Object.keys(geojson).forEach(function (key) {
```

Comment:
Maybe.. but I don't think so?... I think the `cloneAll` feature clones all property values, so if you need to mutate some of the properties then it might be best to use that option.

However.. we could test it out and maybe it can replace the `cloneAll` feature entirely.